about.html

```diff
@@ -0,0 +1,138 @@
+      
+    <link rel="stylesheet" href="./fontawesome/css/all.css" />
+    <link rel="stylesheet" href="about.css" type="text/css">
+    <title>About Capstone</title>
+</head>
+<body>
+    <header>
+        <nav class="black-navbar">
+            <a href="#">
+                <i class="fa-brands fa-facebook-f"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="#">Logout</a>
+            <a href="#">MyPage</a>
+            <a href="#" class="last">Dubai</a>
+        </nav>
+
+        <nav class="white-navbar">
+            <div class="desk-logo">
+                <a href="index.html">
+                    <img src="./images/index logo.png" alt="Index Page Logo">
+                </a>
+            </div>
+            <ul class="nav-itmes">
+              <li>
+                <a class="first" href="about.html">About</a>
+              </li>
+              <li>
+                <a href="#">Program</a>
+              </li>
+              <li>
+                <a href="#">Join</a>
+              </li>
+              <li>
+                <a href="#">Sponser</a>
+              </li>
+              <li>
+                <a href="#">News</a>
+              </li>
+              <li>
+                <a href="#">
+                    <button class="nav-btn">Campaign</button>
+                </a>
+              </li>
+            </ul>
+        </nav>
+        <nav class="mobile-nav">
+          <div class="close-btn">
+              <i class="fa-solid fa-xmark"></i>
+          </div>
+        <ul class="nav-list">
+          <li>
+            <a class="nav-links" href="about.html">About</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Program</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Join</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Sponser</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">News</a>
+          </li>
+          <li>
+            <a href="#">
+                <button class="mobile-nav-btn">Campaign</button>
+            </a>
+          </li>
+        </ul>
+      </nav>  
+        <div class="menu-btn">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+        </div>
+        <div class="about-title">
+            <img src="./images/about_title.png" alt="about title">
+        </div>
+        <div class="red-line"></div>
+        <p class="about-paragraph">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium molestiae tenetur vero consectetur voluptatum sunt debitis nisi, 
+            nemo ullam reprehenderit, sit rem laborum aliquam cupiditate excepturi dolor autem, velit quasi?
+        </p>
+        <p class="contact">
+          Please contact us via email for any further question about CC Global
+          Submit 2015!<br> <a href="#">ccsummit@gmail.com</a>
+        </p>
```

Comment:
- [ ] This is the top section of your home page and would be better placed outside the `<header>` tag and in a proper `<section>` tag. Kindly apply these changes.
 - [ ] Please do not use images to simulate the headings of your page. You should use a `<h1>` tag to wrap the main heading. Please create this heading for this page and try to style as much as possible as this heading you copied.

Reply:
Done